_layouts/partners/faq.html

```diff
@@ -7,86 +7,131 @@
   {% include partners/header.html %}
 </header>
 <script>
+  let accordionButtons = document.getElementsByClassName('login-accordion-btn')
+  let openAccordionCount = 0;
+
+  function updateSingleAccordion(el){
+    setTimeout(function(){
+      if(el.getAttribute("aria-expanded") == 'true'){
+        openAccordionCount++;
+      }else{
+        openAccordionCount--;
+      }
+      toggleExpandAllCollapseAllBtn()
+    }, 10);
+  }
+
+  function toggleExpandAllCollapseAllBtn(){
+    let btns = document.getElementsByClassName('collapse-expand-btn');
+
+    Array.from(btns).forEach(btn => {
+      btn.innerHTML = (openAccordionCount > 0) ? 'Collapse all' : 'Expand all';
+    })
+  }
+
+  function updateAllAccordions(){
+    if(openAccordionCount > 0){
+      closeAllAccordions();
+    }else{
+      openAllAccordions();
+    }
+    toggleExpandAllCollapseAllBtn();
+  }
+
   function closeAllAccordions(){
-    // changes accordion button look through aria controls
-    let accordionButtons = document.getElementsByClassName('usa-accordion__button');
     Array.from(accordionButtons).forEach(el => {
       el.setAttribute("aria-expanded", "false");
     });
-    // hides accordion content panels
-    let accordionContentPanels = document.getElementsByClassName('usa-accordion__content');
+
+    let accordionContentPanels = document.getElementsByClassName('login-accordion-content');
     Array.from(accordionContentPanels).forEach(el => {
       el.hidden = true;
     })
+
+    openAccordionCount = 0;
+  };
+  
+  function openAllAccordions(){
+    openAccordionCount = 0;
+    Array.from(accordionButtons).forEach(el => {
+      el.setAttribute("aria-expanded", "true");
+      openAccordionCount ++;
+    });
+    
+    let accordionContentPanels = document.getElementsByClassName('login-accordion-content');
+    Array.from(accordionContentPanels).forEach(el => {
```

Comment:
Will we support Internet Explorer on the partner subsite?

Reply:
Aside: I was sort of curious if this functionality was something which the design system offered. I see that it's in development, but not yet available.

https://github.com/uswds/uswds/pull/3446

Reply:
I was also curious why the functionality wasn't built in.  It seems pretty important from an accessibility standpoint.

Reply:
As for supporting IE, I think it's still unclear but we will be doing thorough QA on this section and we decided to merge this in to avoid it getting stale and return to it when we have our IE decision.

Reply:
Also, this is just being merged into a dev branch and not into `main` at this point.